virttest/migration.py

```diff
@@ -271,26 +283,196 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
         :param thread_timeout: time out seconds for the migration thread running
         :param ignore_status: determine if an exception is raised for errors
         :param func: the function executed during migration thread is running
+        :param multi_funcs: list of functions executed during migration
+                     thread is running. The func and multi_funcs should not be
+                     provided at same time. For example,
+                     multi_funcs = [{"func": <function check_established at 0x7f5833687510>,
+                                     "after_event": "iteration: '1'",
+                                     "before_event": "Suspended Migrated",
+                                     "before_pause": "yes", "func_name": params},
```

Comment:
func_name should be func_param, right?

Reply:
Updated and thanks for identifying this typo.